### DISCUSSION TITLE
Epic dashboards needed

### DISCUSSION THREAD

**Discussion by totoCZ** [akvorado/akvorado#1581] [Q&A]
Hi @akvorado @ovh

good work on the new project.
One of the main advantages of elastiflow were the pretty (and free) dashboards.
So far I see only the grafana integration being implemented.

1) is it even possible to get the same charting features as ELK with just pure Grafana?
2) it would be nice if your companies could do some work on them/open-source the dashboards

If I still worked at my old employer perhaps I could do the work, but for my own hobby use it seems it could be faster to just get the unmaintained elastiflow working on newer ELK. :) 

![](https://user-images.githubusercontent.com/10326954/89781356-727ede00-db13-11ea-8130-73562a8593f8.png)

**Comment by totoCZ** [comment]
Some more examples
https://edennington.wordpress.com/2020/02/24/netflow-collection-and-visualization-with-elastiflow/

At cdn77 we had our own proprietary system (of course), and I ran this parallel to it. It was quite a good alternative.
Now it's sadly closed source.

I think every internet company decides to re-implement this sFlow stack in one way or another because existing OSS isn't up to par and then it ends up looking about the same.. just with duplicated effort.

So that's why I think we need good OSS dashboards.

**Reply by fatherfbi** [reply]
What's stopping you from making a visualizer? There is a database that already contains all the necessary data.

**Reply by zxyz** [reply]
you could, besides using the newish ovh grafana akvorado plugin, also use the grafana clickhouse plugin to directly query the CH database. grafana offers a lot of visualization plugins, quite sure you'd be able to create some nice dashboards.

**Reply by vincentbernat** [reply]
At some point, I think we will work closer with OVH Grafana plugin: provide a stable API for it to use, maybe ship it in the default install. The current user interface may stay for browsing the data, but won't get more Grafana-like functionalities as this is too much work to maintain.
